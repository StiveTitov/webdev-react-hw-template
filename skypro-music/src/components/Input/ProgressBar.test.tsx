import renderer from "react-test-renderer";
import { ProgressBar } from ".";

describe('Компанент- полосы прогресса', () => {
    it('корректно отрисовывает разметку с начальными значениями', () => {
      // Тестовый код
      const component = renderer.create(<ProgressBar duration={100} currentTime={0} handleSeek={50} />).toJSON();
  expect(component).toMatchSnapshot();
    });
   
    
   });