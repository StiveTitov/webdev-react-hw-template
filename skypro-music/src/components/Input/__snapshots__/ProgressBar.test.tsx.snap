// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компанент- полосы прогресса корректно отрисовывает разметку с начальными значениями 1`] = `
<input
  className="barPlayerProgress"
  max={100}
  min={0}
  onChange={50}
  type="range"
  value={0}
/>
`;
